x-pack/plugins/security_solution/public/explore/links.ts

```diff
@@ -34,6 +34,7 @@ const networkLinks: LinkItem = {
       defaultMessage: 'Network',
     }),
   ],
+  capabilities: [[`${SERVER_APP_ID}.show`, 'ml.canGetMlInfo']],
```

Comment:
If these links are for anomaly detection results then the correct capability to use would be `ml.canGetJobs`.

Reply:
@angorayc 
We should not restrict the top-level (_network_, _users_ and _hosts_) by ML capabilities, they should go to the _Anomalies_ sub-links of each one instead, which are ML-related. 

What about the following:

- Keep ```capabilities: `${SERVER_APP_ID}.show` ``` on these top-level links since they are missing and it makes sense. If this capability is missing sub-links won't be displayed either.

- Add `capabilities: 'ml.canGetJobs', ` to _networkAnomalies_, _usersAnomalies_ and _hostsAnomalies_ sub-links